app/assets/stylesheets/profile.scss

```diff
@@ -31,6 +31,10 @@ a:hover {
   margin-top: 5%;
   background-color: #FF851B;
   color: white;
+  &:hover {
+    background-color: #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`